backend/src/resource/notificacao-secretaria-saude-resource.js

```diff
@@ -0,0 +1,82 @@
+const secretariaApi = require('../services-api/secretaria-saude/service');
+const repos = require('../repositories/repository-factory');
+const tpTransmissaoApiSecretaria = require('../enums/tipo-transmissao-api-secretaria-enum');
+const EnviarNotificacaoRequest = require('../services-api/secretaria-saude/models/enviar-notificacao-request');
+const { RegraNegocioErro } = require('../lib/erros');
+const apiErrors = require('../services-api/secretaria-saude/enums/api-errors');
+
+exports.enviarNotificacao = async (req, res, next) => {
+    try {
+        const { id } = req.params;
+
+        const notificacao = await repos.notificacaoRepository.getPorId(id);
+        const request = new EnviarNotificacaoRequest(notificacao);
+        const response = await secretariaApi.enviarNotificacao(request);
+
+        if (response.statusCode !== 200) {
+            throw new RegraNegocioErro(response.data);
+        }
+
+        return res.json({ data: response });
+    } catch (err) {
+        return next(err);
+    }
+}
+
+exports.getPendentesEnvio = async (req, res, next) => {
+    try {
+        const notificacoes = await repos.notificacaoRepository
+            .getNotificacoesPendentesEnvioSecretaria();
+
+        const notificacoesPendentes = notificacoes.map((data) => {
+            return {
+                notificacaoId: data.Notificacao.id,
+                nomePaciente: data.Notificacao.Pessoa.nome,
+                dataHoraNotificacao: data.dataHoraNotificacao,
+                nomeUnidadeSaude: data.Notificacao.UnidadeSaude.nome
+            }
+        });
+
+        return res.json({ data: notificacoesPendentes });
+    } catch (err) {
+        return next(err);
+    }
+};
+
+exports.enviarNotificacoes = async (req, res, next) => {
+    try {
+        const ids = req.body;
+        const notificacoes = await repos.notificacaoRepository
+            .getNotificacoesPendentesEnvioSecretariaPorIds(ids);
+
+        let dataErrors = [];
+
+        for (var i = 0; i < notificacoes.length; i += 1) {
```

Comment:
Aqui seria melhor criar  um método que executa uma Promise pasando a notificacao e retornando a promisse que fara todas as chamadas.

Da pra usar o Promise.allSettled ele recebe um array de promises e executa todas simultaneamente (ao invés de uma a uma no for) além disso ele retorna o status se foi completado com sucesso e o valor ou se deu erro e o motivo. Segue o link: https://medium.com/trabe/using-promise-allsettled-now-e1767d43e480